server/game/player.js

```diff
@@ -757,7 +757,12 @@ class Player extends Spectator {
                 this.game.addMessage('{0} discards a duplicate to save {1}', this, character);
             }
         } else {
-            this.game.raiseEvent('onCharacterKilled', this, character, allowSave, () => {
+            this.game.raiseEvent('onCharacterKilled', this, character, allowSave, event => {
+                if(character.location !== 'play area') {
+                    event.cancel();
```

Comment:
I did feel a bit icky about this but the alternative seems to be rewriting the entire kill + save system (again). That's on the menu as we move toward first player choosing the order of triggered abilities, but this seemed like a 'good enough' fix in the meantime.